### PR TITLE
resource/alicloud_click_house_db_cluster: Added support for creating multi-zone DBCluster.

### PR DESCRIPTION
**Pull request by itkillsnone**
resource/alicloud_click_house_db_cluster: Added support for creating multi-zone DBCluster.

### CODE REVIEW

File: alicloud/resource_alicloud_click_house_db_cluster.go [hunk 1]
```diff
@@ -152,6 +152,27 @@ func resourceAlicloudClickHouseDbCluster() *schema.Resource {
 				Computed: true,
 				ForceNew: true,
 			},
+			"multi_zone_vswitch": {
+				Type:     schema.TypeMap,
```
[the remark this answers is not in the document]
Comment (itkillsnone) [reply alicloud/resource_alicloud_click_house_db_cluster.go]: 修改成TypeSet了

File: alicloud/resource_alicloud_click_house_db_cluster.go [hunk 2]
```diff
@@ -152,6 +152,25 @@ func resourceAlicloudClickHouseDbCluster() *schema.Resource {
...
 				ForceNew: true,
 			},
+			"multi_zone_vswitch_list": {
+				Type:     schema.TypeSet,
+				Optional: true,
+				ForceNew: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"zone_id": {
+							Optional: true,
```
Comment (ChenHanZhang) [inline alicloud/resource_alicloud_click_house_db_cluster.go:162]: 这里是不是会自动返回zone_id，会的话加上Computed

**Review by ChenHanZhang — COMMENTED** [review]
approved

**Review by ChenHanZhang — COMMENTED** [review]
approved

**Review by ChenHanZhang — COMMENTED** [review]
approved

**Review by ChenHanZhang — COMMENTED** [review]
approved

**Review by ChenHanZhang — COMMENTED** [review]
approved

**Comment by itkillsnone** [comment]
$TF_ACC=1 TF_LOG=INFO go test ./alicloud -v -run=TestAccAliCloudClickHouseDBCluster_multiZone  -timeout=1440m -count=1 
=== RUN   TestAccAliCloudClickHouseDBCluster_multiZone
    provider_test.go:152: Skipping unsupported region eu-central-1. Supported regions: [cn-hangzhou]. Using cn-hangzhou as this test region
--- PASS: TestAccAliCloudClickHouseDBCluster_multiZone (1038.54s)
PASS
ok      github.com/aliyun/terraform-provider-alicloud/alicloud  1038.853s

**Pull request merged**